config/application.rb

```diff
@@ -2,6 +2,8 @@
 
 require 'rails/all'
 
+require 'carrierwave'
```

Comment:
No need to include carrierwave like this! It is beeing done a couple of lines below by the bundler.